apps/jupyter/extensions/notebook-runner/src/__tests__/helpers/exports.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  CrtInternalTask,
+  TaskAssetFiles,
+  GenericNotebookTask,
+} from "../../task-converter";
+
+export const assertSharedDirectoriesEquality = async (
+  actual: TaskAssetFiles,
```

Comment:
Nitpick: Based on my previous comment I would personally prefer consistent terminology, i.e. calling this `assertTaskAssetFilesDirectoriesEquality` or calling the type `SharedDirectories` :)